docs/errors/errors.md

```diff
@@ -2,40 +2,42 @@
 Pydantic will raise a [`ValidationError`][pydantic_core.ValidationError] whenever it finds an error in the data it's validating.
 
 !!! note
-    Validation code should not raise `ValidationError` itself, but rather raise a `ValueError` or
-    `AssertionError` (or subclass thereof) which will be caught and used to populate
-    `ValidationError`.
+    Validation code should not raise the [`ValidationError`][pydantic_core.ValidationError] itself,
+    but rather raise a [`ValueError`][] or a [`AssertionError`][] (or subclass thereof) which will
+    be caught and used to populate the final [`ValidationError`][pydantic_core.ValidationError].
 
-One exception will be raised regardless of the number of errors found, that `ValidationError` will
-contain information about all the errors and how they happened.
+    For more details, refer to the [dedicated section](../concepts/validators.md#raising-validation-errors)
+    of the validators documentation.
```

Comment:
This validators section already covers what was said below about custom errors, so I removed it to avoid duplication